recipes-images/images/torizon-core-common-podman.bb

```diff
@@ -0,0 +1,18 @@
+TDX_COMMON = "1"
+
+require torizon-os-version.inc
+
+SUMMARY = "TorizonCore using Podman"
+DESCRIPTION = "TorizonCore Linux with no containers pre-provisioned \
+using Podman container engine."
+
+require torizon-core-common.inc
+
+IMAGE_VARIANT = "Podman"
+
+CORE_IMAGE_BASE_INSTALL:append = " \
+    podman \
+    podman-compose \
+    fuse-overlayfs \
+    toolbox \
```

Comment:
I don't have much knowledge of `toolbox`. But from what little I've read, it seems like something aimed more at development. Would not it be better to dd `torizon-core-common-dev-podman` and then `toolbox` in this image?

Reply:
Sure, it can be viewed that way. My intention with Toolbox is making the Podman flavour of Torizon more like Fedora Silverblue, but with Yocto behind it.

I'll add a dev podman image to the MR.

Reply:
FWIW, I'd love to see a [distrobox](https://distrobox.it/) option as well.